components/Chat/MessageGroup.vue

```diff
@@ -1,10 +1,8 @@
 <template functional>
   <div class="mb-2">
-    <div v-for="(message, i) in props.group" :key="i">
-      <span :id="`msg:${message.timestamp}`"/>
-      <message :heading="i === 0" darken-on-hover
-        :from="message.from" :body="message.body" :timestamp="message.timestamp"/>
-    </div>
+    <message v-for="(message, i) in props.group" :key="i"
```

Comment:
How would we access this? I was having problems with dynamic refs not showing up on `this.$refs`.

Reply:
See: fffa60e2032e00d385af0816ce9e29276143b9dc 
```
chore: revert "feat(scrollback): add infinite message scrollback"

This reverts commit 862c1b0, reversing
changes made to 0bf5c20.
```
This commit made things too buggy, crashed a lot, and was hard to develop with.